open_flamingo/train/data.py

```diff
@@ -294,32 +294,25 @@ def preprocess_interleaved(sample, tokenizer, clip_processor, sim_threshold):
 
     images, sentence_ixs = [], []
     for sample_image in info["image_info"]:
-        image_base64 = sample_image["image_base64"]
```

Comment:
Can we have an option to pass in shards with raw images OR clip embeddings? This will restrict the types of training runs we can do otherwise